test/src/test/java/org/corfudb/runtime/collections/CorfuStoreTest.java

```diff
@@ -99,11 +102,104 @@ public void basicTest() throws Exception {
                 .setEventTime(fifty)
                 .build();
 
+        assertThat((Message) q.get(tableName, timestamp, lookupKey)).isNull();
```

Comment:
do we have any test that exercises a non existing timestamp?

Reply:
will be coming up. thanks